pt-BR/association_basics.md

```diff
@@ -1,23 +1,28 @@
 **NÃO LEIA ESTE ARQUIVO NO GITHUB, OS GUIAS SÃO PUBLICADOS NO https://guiarails.com.br.**
 **DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
 
-Active Record Associations
+Associações Active Record
 ==========================
 
-This guide covers the association features of Active Record.
+Esse guia cobre os recursos de associação do _Active Record_.
 
-After reading this guide, you will know:
+Após ler esse guia, você saberá:
 
-* How to declare associations between Active Record models.
-* How to understand the various types of Active Record associations.
-* How to use the methods added to your models by creating associations.
+* Como declarar associações entre modelos _Active Record_.
```

Comment:
```suggestion
* Como declarar associações entre *models* do _Active Record_.
```
A gente não tá traduzindo *model*, deve ficar só em itálico. Dá uma conferida no [Manual de tradução](https://github.com/campuscode/rails-guides-pt-BR/blob/master/TRANSLATION_MANUAL.md).